### ISSUE TITLE
[MF v1.0.9; SF-EA-123631] Game Crash when pointing at Door10 with Paint Gun

### ISSUE THREAD

**Issue by xenoxium** [klangzwang/MoarFactory#12]
When Pointing at a Door10 (the automatic opening one) with the Paint gun the game crashes.

Stack-Trace:
```
Version: 123631, IsEditor: No, IsPerforceBuild: No, BuildConfiguration: Shipping, Launcher: Epic, NetMode: Listen Server, IsUsingMods: No

Unhandled exception

FactoryGame_Win64_Shipping!AFGBuildable::GetCanBeColored_Implementation() [d:\ws\sb-160502110050-fad\ue4\games\factorygame\source\factorygame\private\buildables\fgbuildable.cpp:549]
FactoryGame_Win64_Shipping!IFGChainsawableInterface::execIsChainsawable() [d:\ws\sb-160502110050-fad\ue4\games\factorygame\source\factorygame\public\fgchainsawableinterface.h:22]
FactoryGame_Win64_Shipping!UFunction::Invoke() [d:\ws\sb-160502110050-fad\ue4\engine\source\runtime\coreuobject\private\uobject\class.cpp:4644]
FactoryGame_Win64_Shipping!UObject::ProcessEvent() [d:\ws\sb-160502110050-fad\ue4\engine\source\runtime\coreuobject\private\uobject\scriptcore.cpp:1464]
FactoryGame_Win64_Shipping!AActor::ProcessEvent() [d:\ws\sb-160502110050-fad\ue4\engine\source\runtime\engine\private\actor.cpp:825]
FactoryGame_Win64_Shipping!IFGColorInterface::Execute_GetCanBeColored() [d:\ws\sb-160502110050-fad\ue4\games\factorygame\intermediate\build\win64\factorygame\inc\factorygame\fgcolorinterface.gen.cpp:379]
FactoryGame_Win64_Shipping!AFGColorGun::FindIfValidColorTarget() [d:\ws\sb-160502110050-fad\ue4\games\factorygame\source\factorygame\private\fgcolorgun.cpp:151]
FactoryGame_Win64_Shipping!AFGColorGun::Tick() [d:\ws\sb-160502110050-fad\ue4\games\factorygame\source\factorygame\private\fgcolorgun.cpp:84]
FactoryGame_Win64_Shipping!AActor::TickActor() [d:\ws\sb-160502110050-fad\ue4\engine\source\runtime\engine\private\actor.cpp:1020]
FactoryGame_Win64_Shipping!FActorTickFunction::ExecuteTick() [d:\ws\sb-160502110050-fad\ue4\engine\source\runtime\engine\private\actor.cpp:167]
FactoryGame_Win64_Shipping!FTickFunctionTask::DoTask() [d:\ws\sb-160502110050-fad\ue4\engine\source\runtime\engine\private\ticktaskmanager.cpp:290]
FactoryGame_Win64_Shipping!TGraphTask<FTickFunctionTask>::ExecuteTask() [d:\ws\sb-160502110050-fad\ue4\engine\source\runtime\core\public\async\taskgraphinterfaces.h:847]
FactoryGame_Win64_Shipping!FNamedTaskThread::ProcessTasksUntilQuit() [d:\ws\sb-160502110050-fad\ue4\engine\source\runtime\core\private\async\taskgraph.cpp:581]
FactoryGame_Win64_Shipping!FTaskGraphImplementation::WaitUntilTasksComplete() [d:\ws\sb-160502110050-fad\ue4\engine\source\runtime\core\private\async\taskgraph.cpp:1457]
FactoryGame_Win64_Shipping!FTickTaskSequencer::ReleaseTickGroup() [d:\ws\sb-160502110050-fad\ue4\engine\source\runtime\engine\private\ticktaskmanager.cpp:561]
FactoryGame_Win64_Shipping!FTickTaskManager::RunTickGroup() [d:\ws\sb-160502110050-fad\ue4\engine\source\runtime\engine\private\ticktaskmanager.cpp:1517]
FactoryGame_Win64_Shipping!UWorld::Tick() [d:\ws\sb-160502110050-fad\ue4\engine\source\runtime\engine\private\leveltick.cpp:1568]
FactoryGame_Win64_Shipping!UGameEngine::Tick() [d:\ws\sb-160502110050-fad\ue4\engine\source\runtime\engine\private\gameengine.cpp:1366]
FactoryGame_Win64_Shipping!UFGGameEngine::Tick() [d:\ws\sb-160502110050-fad\ue4\games\factorygame\source\factorygame\private\fggameengine.cpp:9]
FactoryGame_Win64_Shipping!FEngineLoop::Tick() [d:\ws\sb-160502110050-fad\ue4\engine\source\runtime\launch\private\launchengineloop.cpp:4024]
FactoryGame_Win64_Shipping!GuardedMain() [d:\ws\sb-160502110050-fad\ue4\engine\source\runtime\launch\private\launch.cpp:168]
FactoryGame_Win64_Shipping!GuardedMainWrapper() [d:\ws\sb-160502110050-fad\ue4\engine\source\runtime\launch\private\windows\launchwindows.cpp:145]
FactoryGame_Win64_Shipping!WinMain() [d:\ws\sb-160502110050-fad\ue4\engine\source\runtime\launch\private\windows\launchwindows.cpp:275]
FactoryGame_Win64_Shipping!__scrt_common_main_seh() [d:\agent\_work\2\s\src\vctools\crt\vcstartup\src\startup\exe_common.inl:288]
kernel32
ntdll
```

**Comment by yesod-github** [comment]
Thanks for reporting @xenoxium. Independently ran into this matter. @klangzwang: Below outlines all doors that reproduce this issue:

![image](https://user-images.githubusercontent.com/26621448/83705489-14012500-a5e3-11ea-96c3-f82976fe6c46.png)

**Comment by yesod-github** [comment]
Fixed as-of v1.1.3.
